package.json

```diff
@@ -45,10 +45,14 @@
     "@wdio/spec-reporter": "^8.11.2",
     "chromedriver": "^114.0.2",
     "copy-webpack-plugin": "^11.0.0",
+    "eslint": "^8.51.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-compat": "^4.2.0",
     "hastscript": "^8.0.0",
     "mocha": "^10.2.0",
     "path-browserify": "^1.0.1",
     "playwright": "^1.35.1",
+    "prettier": "^3.0.3",
```

Comment:
@trustedtomato Were you actually successfully using this in VS Code? I was getting errors with Prettier v3: https://github.com/prettier/prettier-vscode/issues/3007

Looking at other projects I’m using Prettier w/ VS Code in, they’re all still on v2.8.8.

(FWIW, changing the `prettier.prettierPath` setting to use Prettier’s CommonJS entrypoint did work for me, but you don’t have that in the workspace config file you committed here.)

Reply:
Yup, it worked, but I'll take another look then!

Reply:
Looks like this was fixed in the latest release of VS Code (or maybe the Prettier plugin?). So it might have worked fine for you if you are on the insiders or some other preview/unstable release channel.